### PR TITLE
solution MOYO header

### PR DESCRIPTION
**Pull request by Oleksiy0909**
- [DEMO LINK](https://Oleksiy0909.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Oleksiy0909.github.io/layout_moyo-header/report/html_report/)

-[x] Header height is set in 1 place (for the links)
-[x] Content is vertically centered (for any header height)
-[x] CSS is used to show all letters in Uppercase (don't type them in HTML)
-[x] Logo is an image wrapped with a link
-[x] CSS Variable is used for a blue color
-[x] Pseudo-element is used for a blue line below the active link
-[x] Code follows all the [Code Style Rules ❗️](https://github.com/Oleksiy0909/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,12 +11,110 @@
...
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#!"
```
Comment (volodymyr-soltys97) [inline src/index.html:34]:
```suggestion
        href="#"
```

File: src/index.html [hunk 2]
```diff
@@ -11,12 +11,110 @@
...
+        class="logo"
```
Comment (volodymyr-soltys97) [inline src/index.html:35]:
```suggestion
        class="header__link"
```

File: src/index.html [hunk 3]
```diff
@@ -11,12 +11,110 @@
...
+      >
+        <img
+          class="img__logo"
```
Comment (volodymyr-soltys97) [inline src/index.html:38]:
```suggestion
          class="header__logo"
```

File: src/index.html [hunk 4]
```diff
@@ -11,12 +11,110 @@
...
+          src="/src/images/logo.png"
```
Comment (volodymyr-soltys97) [inline src/index.html:39]:
```suggestion
          src="./images/logo.png"
```

File: src/index.html [hunk 5]
```diff
@@ -11,12 +11,110 @@
...
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
```
Comment (volodymyr-soltys97) [inline src/index.html:43]:
```suggestion
      </a>
      
      <nav class="nav">
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```
Comment (volodymyr-soltys97) [inline src/style.css:4]: Move font styles to the html selector

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,81 @@
...
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:5]: Default value, so you can just remove it
```suggestion
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,81 @@
...
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
```
Comment (volodymyr-soltys97) [inline src/style.css:18]: This color is default
```suggestion
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,81 @@
...
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
```
Comment (volodymyr-soltys97) [inline src/style.css:56]:
```suggestion
.is-active::after {
```

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,81 @@
...
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:53]: Create a variable for repeated color and use it everywhere

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,81 @@
...
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-transform: uppercase;
+}
+
+:hover {
```
Comment (volodymyr-soltys97) [inline src/style.css:79]:
```suggestion
.nav__link:hover {
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

**Comment by Oleksiy0909** [comment]
[DEMO LINK](https://oleksiy0909.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://oleksiy0909.github.io/layout_moyo-header/report/html_report/)
-[x] Header height is set in 1 place (for the links)
-[x] Content is vertically centered (for any header height)
-[x] CSS is used to show all letters in Uppercase (don't type them in HTML)
-[x] Logo is an image wrapped with a link
-[x] CSS Variable is used for a blue color
-[x] Pseudo-element is used for a blue line below the active link
-[x] Code follows all the [Code Style Rules ❗️](https://github.com/Oleksiy0909/layout_moyo-header/blob/master/checklist.md)

**Pull request closed without merging**